topic-operator/src/main/java/io/strimzi/operator/topic/Session.java

```diff
@@ -53,9 +58,11 @@ public class Session extends AbstractVerticle {
     private final static String SASL_TYPE_PLAIN = "plain";
     private final static String SASL_TYPE_SCRAM_SHA_256 = "scram-sha-256";
     private final static String SASL_TYPE_SCRAM_SHA_512 = "scram-sha-512";
+    private final static String SASL_TYPE_CUSTOM = "custom";
```

Comment:
This is not used anywhere.

Reply:
I update Session to use it in setSaslConfigs()